polkadot/scripts/build-only-wasm.sh

```diff
@@ -27,11 +23,17 @@ else
   export WASM_TARGET_DIRECTORY=$2
 fi
 
+
+
 if [ -d $WASM_BUILDER_RUNNER ]; then
   export DEBUG=false
   export OUT_DIR="$PROJECT_ROOT/target/release/build"
-  fl_cargo run --release --manifest-path="$WASM_BUILDER_RUNNER/Cargo.toml" \
+  fl_cargo run --release --manifest-path="$WASM_BUILDER_RUNNER/Cargo.toml" --features fast-runtime \
     | grep -vE "cargo:rerun-if-|Executing build command"
 else
-  fl_cargo build --release -p $1
+  fl_cargo build --release -p $1 --features fast-runtime
 fi
+
+echo "ROCOCO_EPOCH_DURATION=$ROCOCO_EPOCH_DURATION checksums"
+
+md5sum $WASM_TARGET_DIRECTORY/*
```

Comment:
```suggestion
```

Not sure what you want to achieve by this?

Reply:
Should have mark it as a draft I was debugging it.